data_mining/tappedout_draft_display.user.js

```diff
@@ -79,15 +85,19 @@ function imageURL(cardName, wasPicked) {
     return `<img src="https://api.scryfall.com/cards/named?format=image&exact=${encodeURI(cardName)}" class="${cssClass}" width="146" height="204" />`
 }
 
-function getNumDrafters() {
-    const options = Array.from(document.getElementsByTagName('option'))
-    return options.map(o => o.text).filter(t => t.includes('Seat')).length / 2
+function getDrafters() {
+    const options = Array.from(document.getElementById('id_mtgo-seat').options)
+    const drafters = options.map(o => o.text).reduce(
```

Comment:
Would this be simpler as a `filter` and then another `map` rather than this reduce?